src/components/citywide-stats/graph-data.ts

```diff
@@ -0,0 +1,121 @@
+import { DataPoint, MetricDetail, MetricStats, YearData } from './types';
+import HistoricStats from '../../data/dist/historic-stats.json';
+
+const detailKeyMap: Record<MetricDetail, keyof MetricStats> = {
+  count: 'count',
+  mean: 'mean',
+  std: 'std',
+  min: 'min',
+  max: 'max',
+  '25%': 'twentyFifthPercentile',
+  '50%': 'median',
+  '75%': 'seventyFifthPercentile',
+};
+
+export function extractMetricData(
+  metricName: keyof YearData,
+  detail: MetricDetail,
+): DataPoint[] {
+  const historicStats = HistoricStats as Record<string, YearData>;
+  return Object.entries(historicStats)
+    .map(([year, yearData]) => {
+      const stats = yearData[metricName];
+      const value = stats ? stats[detailKeyMap[detail]] : undefined;
+      return {
+        year: parseInt(year),
+        value: value ?? 0,
+      };
+    })
+    .filter((d) => d.value !== 0)
+    .sort((a, b) => a.year - b.year);
+}
+
+export const graphConfigs = [
+  {
+    data: extractMetricData('GHGIntensity', '50%'),
+    containerId: 'ghg-intensity-chart',
+    title: 'GHG Intensity',
+    yAxisLabel: 'kg CO2e/sq ft',
+    color: '#e74c3c',
```

Comment:
I'd suggest making a colors constant to share colors across the whole app - consistent colors are an important way to manage a design!